axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/subscription/GrpcBackedQueryUpdateMessage.java

```diff
@@ -49,8 +50,17 @@
      */
     public GrpcBackedQueryUpdateMessage(QueryUpdate queryUpdate, Serializer serializer) {
         this.queryUpdate = queryUpdate;
-        this.payload = new LazyDeserializingObject<>(new GrpcSerializedObject(queryUpdate.getPayload()), serializer);
-        this.metadata = new GrpcMetadata(queryUpdate.getMetaDataMap(), serializer);
+        this.serializedPayload =
```

Comment:
also here can be invoked the primary private constructor

Reply:
Good suggestion :+1: